dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.11
+
+
+WORKDIR /usr/src/app
+
+
+COPY pyproject.toml ./
+
+# Install poetry for package management
+RUN pip install --no-cache-dir poetry
+RUN poetry install
```

Comment:
**suggestion(non-blocking):** See this article on how to build a good dockerfile for poetry (includes caching etc.): https://medium.com/@albertazzir/blazing-fast-python-docker-builds-with-poetry-a78a66f5aed0